examples/one-tailwind/tailwind.config.js

```diff
@@ -8,5 +8,6 @@ export default {
   theme: {
     extend: {},
   },
+  presets: [require("nativewind/preset")],
```

Comment:
btw this should still work automatically, at least i looked at the nativewind preset and just made sure we add the plugins but i'll check again